index.js

```diff
@@ -24,6 +24,13 @@ const Guardian = require('./models/guardian.js')(sequelize, Sequelize);
 //
 
 
+//------------------------jagan------------------------
+
+const Timetable = require('./models/timetable.js')(sequelize, Sequelize);
```

Comment:
Follow pascal notation for 'Timetable' as TimeTable
```suggestion
const Timetable = require('./models/timetable.js')(sequelize, Sequelize);
```